codegens/java-unirest/lib/parseRequest.js

```diff
@@ -3,17 +3,16 @@ var _ = require('./lodash'),
   sanitize = require('./util').sanitize;
 
 /**
- * Encode param except the following characters- [,{,},]
+ * Encode param except the following characters- [,{,},],%
  *
  * @param {String} param
  * @returns {String}
  */
 function encodeParam (param) {
   return encodeURIComponent(param)
     .replace(/%5B/g, '[')
-    .replace(/%7B/g, '{')
     .replace(/%5D/g, ']')
-    .replace(/%7D/g, '}')
+    .replace(/%25/g, '%')
```

Comment:
@VShingala In Java-unirest library, using characters `{` and `}` resulted in `IllegalURLException` being thrown by the library. We caught this now only because we added a Newman test. So we cannot have those characters in the URL. This is not a problem with other Java libraries (OkHttp) so seems like a bug with unirest.